Meadowlark-js/services/meadowlark-fastify/package.json

```diff
@@ -31,6 +31,10 @@
     "build:copy-non-ts": "copyfiles -u 1 -e \"**/*.ts\" \"src/**/*\" dist --verbose",
     "start:local": "node --require=dotenv/config dist/index.js",
     "start:local:ts": "ts-node --require=dotenv/config src/index.ts",
-    "createKey": "openssl rand -base64 256"
+    "createKey": "openssl rand -base64 256",
+
+    "clinic:flame": "FASTIFY_NUM_THREADS=1 clinic flame -- node dist/index.js",
```

Comment:
This is missing the cross-env to set the environment variable.

Given that the clinic dependency is the main package.json shouldn't these scripts be also on the main package.json file?